### PR TITLE
fix: ttl should be in milliseconds

### PR DESCRIPTION
**Pull request by vonovak**
Hello 👋 

one of the major changes in node-cache-manager v5 was changing ttl from seconds to milliseconds: https://github.com/node-cache-manager/node-cache-manager/pull/203

However, this module still works with the ttl as if it were seconds, so if we pass the number `1000` (1 second) as ttl to this module, it will just pass that value to redis which interprets it as 1000 seconds.

This PR fixes this behavior for `set` `mset` and `ttl` functions.

It also makes some improvements to the test suite.

Thank you 🙂

### CODE REVIEW

File: test/index.test.ts [hunk 1]
```diff
@@ -8,8 +8,7 @@ let customRedisCache: RedisCache;
 
 const config = {
   url: 'redis://localhost:6379',
-  ttl: 0,
```
Comment (vonovak) [inline test/index.test.ts:11]: this was confusing, as some test's descriptions make it look like no ttl is applied, where in fact it is defined as `0` here.

However, `0` ttl is broken: https://github.com/redis/node-redis/pull/2349

it's best to just remove it imho 🙂

File: test/index.test.ts [hunk 2]
```diff
@@ -47,28 +46,20 @@ describe('instance', () => {
...
 
 describe('set', () => {
-  it('should store a value without ttl', () =>
-    expect(redisCache.set('foo', 'bar')).resolves.toBeUndefined());
-
-  it('should store a value with a specific ttl', () =>
-    expect(redisCache.set('foo', 'bar', config.ttl)).resolves.toBeUndefined());
+  it('should store a value without ttl', async () => {
+    await expect(redisCache.set('foo', 'bar')).resolves.toBeUndefined();
+    await expect(redisCache.get('foo')).resolves.toBe('bar');
```
Comment (vonovak) [inline test/index.test.ts:51]: I added expectations for getting the stored value out of redis; if the test does not check that, how can we be sure that the `set` call actually worked?

File: test/index.test.ts [hunk 3]
```diff
@@ -47,28 +46,20 @@ describe('instance', () => {
...
+  it('should store a value with a specific ttl', async () => {
+    await expect(redisCache.set('foo', 'bar', 1)).resolves.toBeUndefined();
+    await new Promise((resolve) => setTimeout(resolve, 2));
+    await expect(redisCache.get('foo')).resolves.toBeUndefined();
+  });
 
   it('should not be able to store a null value (not cacheable)', () =>
     expect(redisCache.set('foo2', null)).rejects.toBeDefined());
 
-  it('should store a value without callback', async () => {
```
Comment (vonovak) [inline test/index.test.ts:62]: there are no more "callbacks" so this test no longer makes sense. The same stuff is covered by the tests above

**Review by botika — APPROVED** [review]

**Comment by botika** [comment]
Thanks!

**Pull request merged**